docs/pages/inboxes/manage-inboxes.mdx

```diff
@@ -28,67 +28,43 @@ You can enable a user to add multiple identities to their inbox. Added identitie
 
 You can enable a user to remove an identity from their inbox. You cannot remove the recovery identity.
 
-## Add an identity to an inbox
+## Inbox update and installation limits
 
-:::warning[Warning]
+An inbox ID is limited to 256 inbox updates. Inbox updates include actions like:
 
-This function is delicate and should be used with caution. Adding an identity to an inbox ID B when it's already associated with an inbox ID A will cause the identity to lose access to inbox ID A.
+- Add a wallet
+- Remove a wallet
+- Add an installation
+- Revoke an installation
 
-:::
+This means that one inbox ID can have 256 installations. When you add an inbox ID to a group, you are adding every installation as a member of the group. This means a group with an inbox that has 200+ installations can very quickly make a group size very large, which can have downstream performance implications.
 
-:::code-group
+If an inbox is allowed to reach its limit of 256 inbox updates, the user will need to [rotate their inbox](#rotate-an-inbox-id) to continue using their identity with XMTP. Rotating their inbox will cause the user to lose of their existing conversations, but will enable them to continue using their identity with new XMTP conversations.
 
-```tsx [Browser]
-await client.unsafe_addAccount(signer, true)
-```
+To avoid this scenario altogether, inboxes are limited to having only 5 active installations. When an inbox has 5 active installations, the user must revoke 1 installation before they are able to add another. This helps keep groups to a reasonable size and also helps keep users from accidentally reaching the 256 limit. However, note that revoking a installation counts toward the 256 inbox update limit, so frequently revoking installations and adding new ones can still cause a user to quickly reach the inbox update limit.
 
-```tsx [Node]
-await client.unsafe_addAccount(signer, true)
-```
+### Rotate an inbox ID
 
-```jsx [React Native]
-await client.addAccount(identityToAdd)
-```
+When you create an inbox ID, there is nonce on it that is used to create the ID. If an inbox reaches its inbox updates limit, you can increment the nonce and it will create a new inbox ID associated with the user's information.
```

Comment:
```suggestion
When you create an inbox ID, there is a nonce on it that is used to create the ID. If an inbox reaches its inbox updates limit, you can increment the nonce and it will create a new inbox ID associated with the user's information.
```